src/routes/tools/extreme-heat/_ExploreData.svelte

```diff
@@ -140,11 +142,13 @@
       : $indicator.title;
 
   $: if ($data) {
-    statsData = $data.filter((d) => d.type !== "area");
-    dataByDate = getDataByDate(flattenData($data));
+    if ($indicator.id === "timing") {
+      dataByDate = getDataByDay(flattenData($data));
+    } else {
+      dataByDate = getDataByDate(flattenData($data));
```

Comment:
should these functions be renamed to  `groupDataByYear`  and `groupDataByDay`?

Reply:
@mukhtyar IMO it never hurts to rename something so that it's more semantic and makes the code easier to follow.